sklearn/naive_bayes.py

```diff
@@ -579,7 +579,9 @@ class MultinomialNB(BaseDiscreteNB):
     ----------
     C.D. Manning, P. Raghavan and H. Schuetze (2008). Introduction to
     Information Retrieval. Cambridge University Press, pp. 234-265.
-    http://nlp.stanford.edu/IR-book/html/htmledition/naive-bayes-text-classification-1.html
+
+    http://nlp.stanford.edu/IR-book/html/htmledition/\
```

Comment:
The HTML rendering was [not aligned properly] (http://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.MultinomialNB.html) without this... Should I leave it as such and wait for the fix at numpydoc?